### DISCUSSION TITLE
Can't join on Terraria Mobile 1.4.0.5.2

### DISCUSSION THREAD

**Discussion by DeadRaider37** [Pryaxis/TShock#2285] [Q&A] [answered]
So I use the following things:

Terraria Mobile 1.4.0.5.2

TShock 4.5.2

I can't join the server from Terraria Mobile, and I searched the config, I'm really lost here. It says "You are not using the same version as the server" and the console says:

[IP] is connecting...
[IP] was booted: You are not using the same version as this server.

idk what to do.

**Comment by bartico6** [comment, marked as answer]
TShock 4.5.2 is for Terraria 1.4.2.2.
**NOT** 1.4.0.5(.2)

**Reply by Kojirremer** [reply]
Mobile Terraria is not supported on the official repo. There are forks, but they are maintained by third parties. If you use them, it's at your own risk.

**Reply by bartico6** [reply]
> 
> 
> So it's not compatible with Terraria mobile? 1.4.0.5.2 is the latest version for mobile

its not even intended for it. tshock is for pc terraria not mobile. any trickery you need to do to make it work with mobile comes at your own risk [of running outdated software] and the risk of your players' devices not handling multiplayer trickery appropriately. mobile terraria is a lot more fragile than desktop.

**Reply by hakusaro** [reply]
In general, you can run TShock software by official collaborators like @mistzzt or @AxeelAnder if you can find their work -- I stand by them.
